projects/plugins/jetpack/modules/sharedaddy/sharing.php

```diff
@@ -731,6 +730,16 @@ function ( $carry, $item ) {
 			},
 			false
 		);
+
+		$host = new Status\Host();
+
+		$wpcom_link = 'https://wordpress.com/support/wordpress-editor/blocks/sharing-buttons-block/';
+
+		if ( function_exists( 'localized_wpcom_url' ) ) {
+			$wpcom_link = localized_wpcom_url( $wpcom_link );
+		}
+
+		$link = $host->is_wpcom_platform() ? $wpcom_link : 'https://jetpack.com/support/sharing/';
```

Comment:
Should we keep using `Redirect::get_url` for self-hosted sites? cc @jeherve 

According to PCYsg-pY7-p2:

> Any external link in the Jetpack plugin must be added using the jetpack.com/redirect service.
>
> This allows us to have control over the destination of each link in the plugin and the ability to change it even for users using outdated versions of the plugin.